pkgs/games/ldmud/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib, fetchFromGitHub, stdenv, autoreconfHook, pkg-config, bison, libiconv
+, pcreSupport ? true, pcre, gcryptSupport ? true, libgcrypt, erqSupport ? "erq"
+, compatModeSupport ? false, filenameSpacesSupport ? false, ipv6Support ? false
+, mccpSupport ? false, zlib, mysqlSupport ? false, libmysqlclient
+, postgresSupport ? false, postgresql, sqliteSupport ? false, sqlite
+, jsonSupport ? false, json_c, xmlSupport ? false, libxml2
+, deprecatedSupport ? true, tlsSupport ? false, openssl, pythonSupport ? false
```

Comment:
Do we really need all those options? No one is going to test them in all combinations. How about we make some good default assumptions and drop half of those options?

Reply:
That's fair feedback. I will rework and drop some low hanging fruit like `deprecatedSupport`, and `filenameSpacesSupport`.

Reply:
I've trimmed it down to just:
* MCCP support
* Mysql support
* Postgres support
* Sqlite support
* TLS support
* Python support
* IPv6 support

I dropped the configurable erq mode, the deprecated efuns option, the fielname spaces option, gcrypt option, pcre option, and compat mode option.